### ISSUE TITLE
[completed-docs] False negative on class-level docs if a member of the class has @inheritdoc

### ISSUE THREAD

**Issue by UselessPickles** [palantir/tslint#3806]
### Bug Report

- __TSLint version__: 5.9.1
- __TypeScript version__: 2.7.2
- __Running TSLint via__: CLI

#### TypeScript code being linted

```ts
class Foo {
    /**
     * @inheritdoc
     */
    public foo(): void {
        // empty
    }
}
```

with `tslint.json` configuration:

```json
{
    "extends": [
        "tslint:recommended",
        "tslint-react",
        "tslint-blueprint",
        "tslint-eslint-rules",
        "tslint-config-prettier"
    ],
    "rulesDirectory": [
        "tslint-rules/compiled"
    ],
    "rules": {
        "completed-docs": [
            true,
            {
                "classes": {
                    "tags": {
                        "content": {},
                        "existence": ["inheritdoc"]
                    }
                },
                "enums": {
                    "tags": {
                        "content": {},
                        "existence": ["inheritdoc"]
                    }
                },
                "enum-members": {
                    "tags": {
                        "content": {},
                        "existence": ["inheritdoc"]
                    }
                },
                "functions": {
                    "tags": {
                        "content": {},
                        "existence": ["inheritdoc"]
                    }
                },
                "interfaces": {
                    "tags": {
                        "content": {},
                        "existence": ["inheritdoc"]
                    }
                },
                "methods": {
                    "privacies": ["all"],
                    "tags": {
                        "content": {},
                        "existence": ["inheritdoc"]
                    }
                },
                "namespaces": {
                    "tags": {
                        "content": {},
                        "existence": ["inheritdoc"]
                    }
                },
                "properties": {
                    "privacies": ["all"],
                    "tags": {
                        "content": {},
                        "existence": ["inheritdoc"]
                    }
                },
                "types": {
                    "tags": {
                        "content": {},
                        "existence": ["inheritdoc"]
                    }
                }
            }
        ],
        "valid-jsdoc": [
            true,
            {
                "requireParamType": false,
                "requireParamDescription": true,
                "requireReturn": false,
                "requireReturnType": false,
                "requireReturnDescription": true
            }
        ],
        "typedef": [
            true,
            "call-signature",
            "parameter",
            "property-declaration"
        ],
        "new-parens": true,
        "no-var-requires": false,
        "no-arg": true,
        "no-conditional-assignment": true,
        "no-console": [
            true,
            "debug",
            "info",
            "log",
            "time",
            "timeEnd",
            "trace"
        ],
        "object-literal-sort-keys": false,
        "array-type": [true, "array"],
        "ordered-imports": false,
        "quotemark": [true, "double", "jsx-double", "avoid-escape"],
        "class-name": false,
        "interface-name": false,
        "no-namespace": false,
        "no-default-export": true,
        "no-unused-variable": true,
        "object-literal-shorthand": [true, "never"],
        "max-classes-per-file": [false],
        "triple-equals": [true, "allow-null-check"],
        "no-empty-interface": false,
        "no-string-literal": false,
        "jsdoc-format": true,
        "member-ordering": false,
        "forin": true,
        "no-for-in-array": true,
        "no-invalid-template-strings": true,
        "jsx-no-multiline-js": false,
        "jsx-no-lambda": false,
        "no-untyped-null-initializer": true,
        "no-non-null-assertion": false,
        "variable-name": [
            true,
            "ban-keywords",
            "check-format",
            "allow-leading-underscore",
            "allow-pascal-case"
        ],
        // Working around this issue: https://github.com/palantir/tslint/issues/3021
        "no-shadowed-variable": [
            true,
            {
                "class": false,
                "enum": false,
                "function": false,
                "interface": false,
                "namespace": false,
                "typeAlias": false,
                "typeParameter": false,
                "import": false
            }
        ]
    }
}
```

#### Actual behavior
No rule failures.

#### Expected behavior
Rule failure due to the class missing documentation.

If I change documentation for method `foo()` to have documentation without `@inheritdoc`, then the rule fails due to missing class-level documentation as expected.

**Comment by JoshuaKGoldberg** [comment]
Duplicate of palantir.github.io/tslint/rules/ - rule should certainly respect `@inheritdoc`.

**Comment by UselessPickles** [comment]
It's probably worth clarifying that this isn't specific to `@inheritdoc`, but probably more generically an issue with the `"existence"` feature that I'm using to implement the desired funcionality for `@inheritdoc`. 

It seems that the `"existence"` feature is incorrectly cascading up, such that the class-level documentation validation is waived if any member of the class has its validation waved due to the `"existence"` feature.

**Comment by JoshuaKGoldberg** [comment]
### 💀 _**It's time!**_ 💀

**TSLint is deprecated** and no longer accepting pull requests other than security fixes. See #4534. ☠️ 
We recommend you instead use **[typescript-eslint](https://typescript-eslint.io)** to lint your TypeScript code with **ESLint**. ✅

👋 It was a pleasure open sourcing with you!

**Comment by JoshuaKGoldberg** [comment]
🤖 Beep boop! 👉  **TSLint is deprecated** 👈 _(#4534)_  and you should **switch to [typescript-eslint](https://typescript-eslint.io)**! 🤖 

🔒 This issue is being locked to prevent further unnecessary discussions. Thank you! 👋
